notebooks/experiments/conditional_diffusion/dna_diff_baseline_conditional_UNET.ipynb

```diff
@@ -0,0 +1,7119 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def metric_comparison_between_components(original_data, generated_data, x_label_plot, y_label_plot):
</pre><ul><li>We should add the <code>Dict</code> type hint to <code>original_data</code> and <code>generated_data</code>.</li><li>If I understand correctly <code>x_label_plot</code> and <code>y_label_plot</code> are strings (type: <code>str</code>) ? If yes we should add that.</li><li>We should add the <code>None</code> type hint to the function output.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pinellolab/DNA-Diffusion/pull/60/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>